src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__staff_renewal_feed.sql

```diff
@@ -0,0 +1,98 @@
+with
+    step_scales as (

```

Comment:
I think this could be the main `select` statement?

Reply:
not sure what you mean here. Can you clarify?